rskj-core/src/main/java/co/rsk/mine/MinerUtils.java

```diff
@@ -48,9 +48,14 @@
 import java.util.function.Function;
 
 public class MinerUtils {
-
     private static final Logger logger = LoggerFactory.getLogger("minerserver");
 
+    private final SignatureCache signatureCache;
```

Comment:
hmm.. this is a utility class, so ideally without any state, and all its methods should should be static